modules/api/test/TestPlan.md

```diff
@@ -33,10 +33,11 @@ See specification for the expected response format.
 
 
 #### Candidate creation - POST /candidates
-Candidate creation
-This is done by sending a POST request to /candidates with the required data. Note the usage of the Authorization header.
+This is done by sending a POST request to /candidates with the required data. Note the usage of the Authorization header. For a candidate to be created, the candidate's informations must also be included. The following command contains the minimal informations necessary the the request to create a new candidate:
+
 ```bash
-~$ curl -H "Authorization: Bearer $token" https://<your-hostname>/api/v0.0.3/candidates -d '{see specs for content}'
+~$ curl -H "Authorization: Bearer $token" https://<your-hostname>/api/v0.0.3/candidates -d '{"Candidate":{"Project":"Rye","Site":"Montreal","DoB":"2019, "Sex":"Female"}}' 
```

Comment:
add a note saying - also see Specs for format instructions
```suggestion
~$ curl -H "Authorization: Bearer $token" https://<your-hostname>/api/v0.0.3/candidates -d '{"Candidate":{"Project":"Rye","Site":"Montreal","DoB":"2019, "Sex":"Female"}}' 
```
```suggestion
~$ curl -H "Authorization: Bearer $token" https://<your-hostname>/api/v0.0.3/candidates -d '{"Candidate":{"Project":"Rye","Site":"Montreal","DoB":"2019", "Sex":"Female"}}' 
```
i think you were missing a "